models/DataObject/Concrete/Dao.php

```diff
@@ -187,17 +187,19 @@ public function update(bool $isUpdate = null): void
         $db = Db::get();
 
         foreach ($fieldDefinitions as $fieldName => $fd) {
-            if ($fd instanceof LazyLoadingSupportInterface && $fd->getLazyLoading()) {
-                if (!$this->model->isLazyKeyLoaded($fieldName) || $fd instanceof DataObject\ClassDefinition\Data\ReverseObjectRelation) {
-                    //this is a relation subject to lazy loading - it has not been loaded
-                    $untouchable[] = $fieldName;
+            if($fd instanceof DataObject\ClassDefinition\Data\Relations\AbstractRelations) {
```

Comment:
Could you please share some insights behind this change?

By following the `$untouchable` var usages, ended up in this case in URL Slug data type

https://github.com/pimcore/pimcore/blob/8d64417b3cc8c9b1b3f2ab25161b761f45a14780/models/DataObject/ClassDefinition/Data/UrlSlug.php#L176-L178

Before PR it's `true`, after PR is always `false`, not sure if there was something legacy leftover but looks like a potential behavior change